ci/test/00_setup_env.sh

```diff
@@ -21,6 +21,9 @@ echo "Fallback to default values in env (if not yet set)"
 export MAKEJOBS=${MAKEJOBS:--j4}
 # A folder for the ci system to put temporary files (ccache, datadirs for tests, ...)
 export BASE_SCRATCH_DIR=${BASE_SCRATCH_DIR:-$BASE_ROOT_DIR/ci/scratch/}
+# What host to compile for. See also ./depends/README.md
+# Tests that need cross-compilation export the appropriate HOST.
+# Tests that run natively, do not set a HOST, but we assume x86_64 here if the calling environment did not set a HOST
 export HOST=${HOST:-x86_64-unknown-linux-gnu}
```

Comment:
What is this host define used for?
Could this be auto-detected instead of assumed?

Reply:
The host is passed down to depends and then used there to cross compile. Also, it is used to name the distdir folder.

Reply:
So if you are not using depends, the host is only used to name the distdir folder.

Reply:
So I am not sure if it makes sense to add auto-detect code to pick the right temporary name

Reply:
It does look too much like "hardcoding an architecture" to me, which the point of this PR is to avoid. If it's only used to name a folder, can't we name it something architecture-independent?

Reply:
Fixed